substrate/frame/vesting/src/lib.rs

```diff
@@ -464,6 +463,14 @@ pub mod pallet {
 }
 
 impl<T: Config> Pallet<T> {
+	// Public function for accessing vesting storage
+	pub fn vesting(
+		account: T::AccountId,
+	) -> Option<BoundedVec<VestingInfo<BalanceOf<T>, BlockNumberFor<T>>, MaxVestingSchedulesGet<T>>>
+	{
+		Vesting::<T>::get(account)
+	}
```

Comment:
If you create this function, why not use it inside the pallet as well?

Reply:
Actually I implement this public function  for PR to be 100% backwards compatible (for outside dependencies too).
I can use this function inside the pallet also but to skip thing easy understandable I haven’t. Also shawn mentioned  
> I have also found it easier to audit storage access by searching for StorageName::<T>::